github-push/handler.go

```diff
@@ -100,14 +103,14 @@ func Handle(req []byte) string {
 		}
 
 		audit.Post(auditEvent)
+
+		status.AddStatus(sdk.StatusFailure, msg, sdk.StackContext)
+		reportStatus(status)
 		return msg
 	}
 
 	serviceValue := sdk.FormatServiceName(pushEvent.Repository.Owner.Login, pushEvent.Repository.Name)
 
-	eventInfo := sdk.BuildEventFromPushEvent(pushEvent)
```

Comment:
Why was this removed?

Reply:
OK it was moved up. Can you confirm that the original path still works fine? i.e. "pending status" is sent?

Reply:
It's moved up before the check as we need the `status` object

Reply:
I will test the change and post comment underneath